yarn.lock

```diff
@@ -7556,7 +7574,7 @@ yargs@^8.0.2:
     y18n "^3.2.1"
     yargs-parser "^7.0.0"
 
-yargs@^9.0.0, yargs@^9.0.1:
+yargs@^9.0.0:
```

Comment:
Why do we still have `yargs@9`? Did you delete and recreate yarn.lock after updating the package.json files?

Reply:
I did - turns out we have a couple of yargs in our dependency tree..:

```
$ yarn list yargs
├─ flow-typed@2.2.1
│  └─ yargs@4.8.1
├─ jest-cli@21.2.1
│  └─ yargs@9.0.1
├─ jest-runtime@21.2.1
│  └─ yargs@9.0.1
├─ lerna@2.4.0
│  └─ yargs@8.0.2
├─ uglify-js@2.8.29
│  └─ yargs@3.10.0
├─ webpack-dev-server@2.9.3
│  └─ yargs@6.6.0
├─ webpack@3.8.1
│  └─ yargs@8.0.2
└─ yargs@10.0.3
```